extensions/requester/rest/requester.rb

```diff
@@ -229,9 +229,13 @@ def request2hash(http)
         def response2hash(http)
           response_data = ''
 
-          if !http.response_data.nil? && (http.response_data.length > (1024 * 100)) # more thank 100K
-            response_data = http.response_data[0..(1024 * 100)]
-            response_data += "\n<---------- Response Data Truncated---------->"
+          if !http.response_data.nil?
+            if (http.response_data.length > (1024 * 100)) # more thank 100K
+              http.response_data = http.response_data[0..(1024 * 100)]
+              http.response_data += "\n<---------- Response Data Truncated---------->"
+            end
+
+            response_data = http.response_data
           end
```

Comment:
I don't think it is a good idea to modify the `http.response_data` property directly. Instead:

```suggestion
          unless http.response_data.nil?
            if (http.response_data.length > (1024 * 100)) # more than 100K
              response_data = http.response_data[0..(1024 * 100)]
              response_data += "\n<---------- Response Data Truncated---------->"
            else
              response_data = http.response_data
            end
          end
```